tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     // Default
-    "target": "es5",
+    "target": "es2015",
```

Comment:
Not disagreeing with this change, but just wondering what you ran into that needed it?

Reply:
Nevermind.  I see your explanation in the PR description.  😃

Reply:
@greinard I reverted this. I was using Set initially but moved away from it when moving to use Map.